lib/peek/views/delayed_job.rb

```diff
@@ -2,6 +2,11 @@ module Peek
   module Views
     class DelayedJob < View
       def initialize(options = {})
+        @label = options.delete(:label)
```

Comment:
Minor: mind using `options[:label]` instead? No need to modify the `options` object if we don't need to.